### PR TITLE
golint on daemon/network package

### PR DESCRIPTION
**Pull request by WeiZhang555**
This PR is part of #14756

Signed-off-by: Zhang Wei zhangwei555@huawei.com

### CODE REVIEW

File: daemon/network/settings.go [hunk 1]
```diff
@@ -2,11 +2,13 @@ package network
 
 import "github.com/docker/docker/pkg/nat"
 
+// Address represents IP address
```
Comment (cpuguy83) [inline daemon/network/settings.go]: `Address represents an IP address`

File: daemon/network/settings.go [hunk 2]
```diff
@@ -2,11 +2,13 @@ package network
...
 type Address struct {
 	Addr      string
 	PrefixLen int
 }
 
+// Settings hold daemon network setting information
```
Comment (cpuguy83) [inline daemon/network/settings.go]: `Settings stores configuration details about the daemon network config`

Comment (WeiZhang555) [inline daemon/network/settings.go]: Thank you for suggestion, will change as you say :)

**Comment by duglin** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

File: daemon/network/settings.go [hunk 3]
```diff
@@ -2,11 +2,13 @@ package network
 
 import "github.com/docker/docker/pkg/nat"
 
+// Address represents an IP address
```
Comment (stevvooe) [inline daemon/network/settings.go]: This isn't really sufficient. It should provide information about why this type exists, why we've opted away from using standard types (ie `net.IP`). It is also missing documentation for the fields.

**Comment by stevvooe** [comment]
This package needs another pass. These comments aren't providing any value.

**Pull request merged**